### ISSUE TITLE
Extract book's title and other info from Cover

### ISSUE THREAD

**Issue by GrayFlash** [AOSP-NIT-Surat/Bookstore-ML#1]
Take a sample image, perform the following tasks:

- [x] extract text on its cover - assigned to @naitik6803
- [ ] decide a generic logic to separate Title, Author name, Publication.

We can accept a Jupyter Notebook or Python script for this issue, focus is on the logic contributor comes up with.

**Comment by Naitik6803** [comment]
i will like to do this

**Comment by GrayFlash** [comment]
@Naitik6803 feel free to ask for help with any blocker you might be facing.

Also you can make 2 seperate PR progressively for the issue.

**Comment by Naitik6803** [comment]
I've used EasyOcr library for text recognition part so is it ok? Should I send the PR?

**Comment by GrayFlash** [comment]
Yes sure if it does not throw a error. Just never commit a red code.

**Comment by Naitik6803** [comment]
We don't have any dataset from where we can see if the particular part of text is author or not so I don't think we'll be able to get author's name, instead one thing we can do is to compare the text written on each book and make groups on the basis of most common text

**Comment by GrayFlash** [comment]
Do you mean we should classify each word we obtain into the following categories: `Book Name`, `Author`, `Publisher`, etc?? Some outliners can be:
- Where the title of Book is a persons name, in case of biographies.

If yes, feel free to try it, would love to see the results. Also, one approach I thought of was to simply compare the ratio of size for the text we have obtained. It is based on the observation that the title of a book is usually bigger than the name of author or the publisher. But this will require us to obtain the bounding box around words, and has outliners such as:
- The books with some part of title smaller than the other.
- Books with Title, Author, Some catchy phrase, testimonial or some other stuff written on the cover.

**Comment by Naitik6803** [comment]
we can compare all the strings we got, like if two books have 'c++' written on them then we can group them together, 
what should i do finally then?

**Comment by Naitik6803** [comment]
most of the time book title has largest font on the book cover so yeah we can get the book name by checking y parameters of the fonts, but it's difficult to get author name like that couz it's position is not fixed on the cover, so one thing we can do is just compare with other books for that.
